virttest/libvirt_cgroup.py

```diff
@@ -134,14 +135,29 @@ def __get_standardized_cgroup1_info(self, virsh_cmd=None):
                           the blkio related cgroup info will be returned
         :return: A dict containing the cgroup info
         """
+
+        def __get_cg_file_path(cg_key, cg_path, cg_file_name):
```

Comment:
Sounds like that parameter arrangement as cg_path, cg_file_name,cg_key may be more native

Reply:
the 'cg_key' is mapping to the keys in constant dicts CGROUP_V1_BLKIO_FILE_MAPPING, GROUP_V1_SCHEDINFO_FILE_MAPPING.
It actually represents the controller's category. Should be used as the first param.  thx